ntbs-integration-tests/ServiceDirectory/ServiceDirectorySearchTest.cs

```diff
@@ -234,16 +235,51 @@ private static int GetNumberOfDisplayedResults(IHtmlDocument pageContent)
         private async Task<int> GetTotalExpectedResults(string searchKeyword)
         {
             var searchKeywords = new List<string> {searchKeyword};
+            var context = _factory.Services.GetService<NtbsContext>()!;
             
-            var referenceDataRepository = _factory.Services.GetService<IReferenceDataRepository>()!;
-            var userSearchService = _factory.Services.GetService<IUserSearchService>()!;
-            
-            var expectedRegionResults = (await referenceDataRepository.GetPhecsBySearchKeywords(searchKeywords)).Count;
-            var expectedServiceResults = (await referenceDataRepository.GetActiveTBServicesBySearchKeywords(searchKeywords)).Count;
-            var expectedHospitalResults = (await referenceDataRepository.GetActiveHospitalsBySearchKeywords(searchKeywords)).Count;
-            var expectedUserResults = (await userSearchService.OrderQueryableAsync(searchKeywords)).Count;
+            var expectedRegionResults = await context.PHEC
+                .CountAsync(x => x.Name.ToLower().Contains(searchKeyword));
+            var expectedServiceResults = await context.TbService
+                .Include(t => t.PHEC)
```

Comment:
These `Include` calls aren't doing anything - `Include` only affects what navigation properties are returned from the database, but here the only thing being returned is an integer (i.e. the count) so you would never need to include anything